library/Netboximport/Api.php

```diff
@@ -27,6 +28,12 @@ private function apiRequest($method, $url, $get_params) {
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
         curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
 
+        // If ssl_verify is set, check ssl certificate from netbox host
+        if ($this->ssl_verify === "0") {
+            curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, false);
```

Comment:
I am not sure about this. Certificates for HTTPS are free and plentiful these days. You can even get them for air gapped machines using DNS validation. If you need to use an internal CA, that can be added to the certificate store of the icinga machine or PHP.

Which case or setup makes it necessary to skip certificate validation entirely?

Reply:
I think it's okay to have this option - a lot of software at least offer this option - as long as we keep "sane defaults" (i.e. ssl_verify ON) i see no problem here.

Reply:
I renamed the option. Now its "Ignore SSL Verification". The default is false ;)